pages/chat/[[...roomId]].js

```diff
@@ -195,10 +204,69 @@ export default function Chat() {
                         </ServiceTable.Body>
                     </ServiceTable>
                 </details>
-                <br />
             </DefaultLayout.Sidebar>
             {roomId && (
                 <DefaultLayout.IframeWrapper>
+                    <DefaultLayout.IframeHeader>
+                        {matrix?.rooms?.get(roomId) ? (
+                            <h2>{matrix?.rooms?.get(roomId)?.name}</h2>
+                        ) : (
+                            roomId === 'new' && <h2>{t('new chat')}</h2>
+                        )}
+                        <DefaultLayout.IframeHeaderButtonWrapper>
+                            {matrix?.rooms?.get(roomId) && (
+                                <>
+                                    {' '}
+                                    <InviteUserToMatrixRoom
+                                        roomId={roomId}
+                                        trigger={
+                                            <TextButton
+                                                variant="ghost"
+                                                title={t('Invite users to {{name}}', { name: matrix.rooms.get(roomId).name })}
+                                            >
+                                                <Icon>
+                                                    <RiUserAddLine />
+                                                </Icon>
+                                            </TextButton>
+                                        }
+                                    />
+                                    <CopyToClipboard title={t('Copy chat link to clipboard')} content={'chat/' + roomId} />
```

Comment:
This copies `chat/<roomId>` to my clipboard... shouldn't this be a working link that starts with "https://" so the user can send this to someone else?

Reply:
I guess another option would be to just copy the room ID in the "Matrix style format" of `!3453543tjjrea3123:matrix.org`... at least when pasting this into a Matrix chatroom it would also just work.

Or build a link using the `matrix.to` service, like:
https://matrix.to/#/!hqcBkCRsdsrSZplHmV:medienhaus.dev/$yg6SZ6nsISgSf5kns7WztQiAVIs_myxwyXw1XVPfAtg?via=medienhaus.dev for example.